CHANGELOG.md

```diff
@@ -33,6 +33,7 @@ We use the following categories for changes:
 - The Promscale extension is now required, while the Timescaledb extension remains optional. The minimum Timescaledb version supported is now 2.6.1 [#1132], [#1297]
 - Disable running Promscale in HA and read-only simultaneously [#1254]
 - Metric tables and views are now owned by prom_admin [#1283]
+- HTTP OPTION verb is no longer supported on `/healthz` endpoint because HTTP router was changed in [#1224].
```

Comment:
I would even phrase this the other way around: The new behaviour is that _only_ GET is allowed. Previously HEAD/OPTIONS/PUT/POST would all work.

Reply:
I believe all paths are affected not only `/healthz`